### ISSUE TITLE
AMP Recaptcha not rendered on the Page

### ISSUE THREAD

**Issue by deepakkhetawat** [ampproject/amphtml#29193]
Hi AMP Team,

I have a usecase to use Form including Google Recaptcha. 
For Form I used amp-form-0.1.js and for recaptcha used amp-recaptcha-input-0.1.js.

This is code used:
 	

However on the page form is rendered, but recaptcha is not getting rendered. Can someone please help to fix this bug.

**Comment by caroqliu** [comment]
Hello, thanks for filing this issue! Can you provide an example page or code sample to look at? Have you tried using the [`amp-recaptcha-input` example](https://amp.dev/documentation/examples/components/amp-recaptcha-input/#basic-usage) on amp.dev?

**Comment by deepakkhetawat** [comment]
`
                            <form id="footer_form_manage_subscriptions" name="footer_form_manage_subscriptions" action-xhr="valid-actiom" method="post" target="_top">
											<div class="input-group">
												<input type="text" aria-labelledby="email-Id" class="form-control emailinput" name="Email" placeholder="yourname@company.com" pattern=".+@.+\..+" value="" required>
                                                <span class="input-group-btn">
                                                    <button type="submit" class="btn btn-subscribe">Subscribe</button>
                                                </span> 
                                            </div>     
											<div submit-success>
												<template type="amp-mustache">
													<p class="form-submit-response"> Successfully Subscribed! Please check your inbox for more details </p>
												</template>
											</div>
											<div submit-error>
												<template type="amp-mustache">
													<p class="form-submit-response"> Failed to Subscribe! </p>
												</template>
											</div>
                                            <amp-recaptcha-input layout="nodisplay" name="recaptcha_example" data-sitekey="valid-site-key" data-action="recaptcha_example">
                                            </amp-recaptcha-input>
										</form>`

Yes, I used amp-recaptcha-input example on amp.dev but i dont see captcha rendered neither in example nor in my solution. Above is the code snippet, please help to fix the same.

**Comment by caroqliu** [comment]
What do you mean by "see recaptcha"? Can you provide a screenshot of what you are receiving and possibly also what you are expecting instead?

**Comment by caroqliu** [comment]
This appears to be a dupe of #30133, closing.
